### PR TITLE
Fix logpoints bug

### PR DESCRIPTION
**Pull request by bomsy**
#### Issues
* The log points are shown as conditional breakpoints
* On editing the log point nothing shows up in the conditional breakpoint panel

logpoints1

* The context menu for a log point should not show any item relating to conditional breakpoints and vice versa
screenshot 2019-01-25 at 21 51 05

### Summary of Changes

* change  log points color to purple
* cleanup unused menu items
* fix `logValue`

### Test Plan

Tell us a little a bit about how you tested your patch.

Example test plan:

- [x] about:config
- [x] set `devtools.debugger.features.log-points` to `true`
- [x] right click editor gutter menu

### Screenshots/Videos (OPTIONAL)

logpoints3

### CODE REVIEW

**Comment by jasonLaster** [comment]
Nice. I like this!

**Comment by darkwing** [comment]
Awesome @bomsy !  A few things I noticed:

1.  Create a conditional breakpoint still shows purple:

conditionalcolor

2.  In this PR it would also be sweet if we could fix the context menu for the breakpoints pane; we still show "Add condition":

conditioncontext

It seems like it fits in with your other context menu fix.

3.  Test failure looks legit, I think because we removed the context menu item.

**Comment by bomsy** [comment]
Thanks for reviewing @darkwing. Will update asap.

**Comment by jasonLaster** [comment]
Okay, this ux is definitely too vague... I took a stab at handling a couple of cases:

1. context menus (gutter, column, panel)
2. marker colors
3. panel bp text

The heuristics I used were:

1. treat logs and conditions as independent (can be added / removed / edited)
1. give precedence to logs (highlight purple, show bp text)

### Adding a new bp
screen shot 2019-02-10 at 9 35 44 am

### with a condition
screen shot 2019-02-10 at 9 36 20 am

### with a log
screen shot 2019-02-10 at 9 37 47 am

### with both set

screen shot 2019-02-10 at 9 38 02 am

File: assets/panel/debugger.properties [hunk 1]
```diff
@@ -534,6 +534,13 @@ editor.addLogPoint.accesskey=l
 # LOCALIZATION NOTE (editor.editLogPoint): Editor gutter context menu item
 # for editing a log point already set on a line.
 editor.editLogPoint=Edit log
+editor.editLogPoint.accesskey=e
```
Comment (flodolo) [inline assets/panel/debugger.properties]: `E` (better to respect the case)

File: assets/panel/debugger.properties [hunk 2]
```diff
@@ -534,6 +534,13 @@ editor.addLogPoint.accesskey=l
...
+
+
+# LOCALIZATION NOTE (editor.removeLogPoint): Context menu item for removing
+# a log point on a line.
+editor.removeLogPoint=Remove log
```
Comment (flodolo) [inline assets/panel/debugger.properties]: nit: can we call this `editor.removeLogPoint.label`?

File: assets/panel/debugger.properties [hunk 3]
```diff
@@ -534,6 +534,13 @@ editor.addLogPoint.accesskey=l
...
+editor.removeLogPoint.accesskey=p
```
Comment (flodolo) [inline assets/panel/debugger.properties]: Why the external accesskey (not available in the label)?

Comment (jasonLaster) [reply assets/panel/debugger.properties]: i don't follow?

Comment (flodolo) [reply assets/panel/debugger.properties]: Label "Remove log", there's no "p", which will result in "Remove log (p)" (with p underscored) on Windows and Linux.

**Pull request merged**